larray/core/array.py

```diff
@@ -2444,6 +2444,13 @@ def __str__(self):
     def __iter__(self):
         return LArrayIterator(self)
 
+    def __contains__(self, key):
+        try:
```

Comment:
I wonder if something like return any(key in axis for axis in self.axes) wouldn't be better.
* "duplicate_label in a" would return True --> better
* "slice('a0', 'a2') in a" would return False --> unsure it's better, but it mimicks lists and np.array